app/src/androidTest/java/com/github/ybecker/epforuml/NewQuestionTest.kt

```diff
@@ -467,10 +469,14 @@ class NewQuestionTest {
         onView(withId(R.id.write_reply_box)).perform(typeText(answerText)).perform(closeSoftKeyboard())
         onView(withId(R.id.post_reply_button)).perform(click())
 
+        Thread.sleep(2000)
+
         //get title name
         val usernameText: ViewInteraction = onView(withId(R.id.qdetails_question_username))
         val text = getText(usernameText).removeSuffix(" asks :")
 
+        Thread.sleep(2000)
```

Comment:
Try to use less Thread.sleep, we should ask the assistants why it is not working when we remove the thead.sleep